microhapulator/api.py

```diff
@@ -89,16 +90,18 @@ def interlocus_balance(
             data.to_csv(tfile, index=False, header=False)
             run(["termgraph", tfile])
     if tofile:
+        backend = matplotlib.get_backend()
+        plt.switch_backend("Agg")
         plt.figure(figsize=figsize, dpi=dpi)
         x = range(len(data))
         y = [c / 1000 for c in data.ReadCount]
-        plt.bar(x, y, color=color)
+        plt.bar(x, y)
         plt.xticks([])
```

Comment:
I think some of these changes are related to the `fix/plots` branch. #131 will need to be merged before this PR is reviewed.